lib/commands/connect/shared.js

```diff
@@ -182,6 +182,19 @@ exports.withConnection = function(context, heroku) {
   });
 }
 
+var withMapping = exports.withMapping = function(connection, object_name) {
+  return Q.Promise(function(resolve, reject) {
+    var found = false, object_name_lower = object_name.toLowerCase();
+    connection.mappings.forEach(function(mapping) {
+      if (mapping.object_name.toLowerCase().indexOf(object_name_lower) == 0) {
```

Comment:
not fetching the connection would be an optimization.  I’ll admit I’m not clear on a use case but more of a hey, it might be useful one day so I’m certainly not blocking the PR on it